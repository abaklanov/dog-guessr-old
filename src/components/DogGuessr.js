import React from "react";

class DogGuessr extends React.Component {
  constructor(props) {
    super(props);

    this.state = {
      photos: []
    };
  }

  render() {
    return "";
  }
}

export default DogGuessr;
