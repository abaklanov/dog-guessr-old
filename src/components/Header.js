import React from "react";
import { Navbar } from "react-bootstrap";
import { FontAwesomeIcon } from "@fortawesome/react-fontawesome";
import { faGithub } from "@fortawesome/free-brands-svg-icons";

class Header extends React.Component {
  render() {
    return (
      <Navbar bg="light" className="justify-content-between">
        <Navbar.Brand href="/">DogGuessr</Navbar.Brand>
        <a
          href="https://github.com/abaklanov/dog-guessr"
          style={{ color: "black" }}
        >
          <FontAwesomeIcon icon={faGithub} />
        </a>
      </Navbar>
    );
  }
}

export default Header;
