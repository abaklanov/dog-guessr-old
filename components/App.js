import React from "react";
import Header from "./Header";
import DogGuessr from "./DogGuessr";
import Footer from "./Footer";

class App extends React.Component {
  render() {
    return (
      <div>
        <Header />
        <DogGuessr />
        <Footer />
      </div>
    );
  }
}

export default App;
