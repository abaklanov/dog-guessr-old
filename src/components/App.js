import React from "react";
import { render } from "react-dom";
import Header from "./Header";
import DogGuessr from "./DogGuessr";

class App extends React.Component {
  render() {
    return (
      <div>
        <Header />
        <div id="title">Dog guessr</div>
        <DogGuessr />
        <div id="photo" />
      </div>
    );
  }
}

export default App;
