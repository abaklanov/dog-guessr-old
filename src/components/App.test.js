import React from "react";
import { shallow } from "enzyme";
import App from "./App";
import Header from "./Header";
import DogGuessr from "./DogGuessr";

const app = shallow(<App />);

it("renders correctly", () => {
  expect(app).toMatchSnapshot();
});

it("renders the Header component", () => {
  expect(app.find(Header)).toHaveLength(1);
});

it("renders the DogGuessr component", () => {
  expect(app.find(DogGuessr)).toHaveLength(1);
});

it("displays the app name `Dog guessr` in a div with id `title`", () => {
  const div = app.find("div#title");
  expect(div).toHaveLength(1);
  expect(div.text()).toEqual("Dog guessr");
});
