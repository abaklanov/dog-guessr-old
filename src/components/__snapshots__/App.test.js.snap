// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
<div>
  <Header />
  <div
    id="title"
  >
    Dog guessr
  </div>
  <DogGuessr />
  <div
    id="photo"
  />
</div>
`;
